### ISSUE TITLE
Adapter conversion to JS

### ISSUE THREAD

**Issue by guybedford** [bytecodealliance/jco#116]
If we had the capability to convert adapter core modules that only do transcoding (eg in https://github.com/bytecodealliance/jco/pull/79) and other simple binding attachment operations into the associated JS code, this would unblock some composition use cases.

We already covert Wasm to JS via `jco transpile --js` but that doesn't support multiple memories. Figuring out the core module subset this conversion applies to would be the first step here I think to determine what structure of JS needs to be output (all Wasm ops, or just a subset?).

**Comment by guybedford** [comment]
Superseded by https://github.com/bytecodealliance/jco/pull/125.
